src/Config/SDK/Configuration/ComponentPlugin.php

```diff
@@ -2,20 +2,8 @@
 
 declare(strict_types=1);
 
-namespace OpenTelemetry\Config\SDK\Configuration;
+namespace OpenTelemetry\API\Configuration\Config;
```

Comment:
This might need a code comment pointing out that the namespace is deliberate, else somebody might "fix" it in the future?